cabal.project

```diff
@@ -1,6 +1,7 @@
 packages:
   cooked-validators
   examples
+  examples/archive/bettinggame
```

Comment:
Maybe we shouldn't add the archive as a buildable package

Reply:
Ideally, we would disable building contracts in the archive by default. However, removing it from here prevents me from building it at all with `cabal-install`. Is there anything we could do strike the balance?

Reply:
hmm... I see. That is unfortunate.